### PR TITLE
cmd/tailscale/cli: update docs and unhide configure

### PR DESCRIPTION
**Pull request by maisem**
Also call out Alpha.

Updates #7220

### CODE REVIEW

File: cmd/tailscale/cli/configure-kube.go [hunk 1]
```diff
@@ -26,12 +26,16 @@ func init() {
...
 	ShortUsage: "kubeconfig <hostname-or-fqdn>",
 	LongHelp: strings.TrimSpace(`
-Run this command to configure your kubeconfig to use Tailscale for authentication to a Kubernetes cluster.
+*** ALPHA; all of this is subject to change ***
+
+Run this command to configure kubectl to connect to a Kubernetes cluster over Tailscale.
 
 The hostname argument should be set to the Tailscale hostname of the peer running as an auth proxy in the cluster.
+
+See: https://tailscale.com/kb/1236/kubernetes-operator/
```
Comment (DentonGentry) [inline cmd/tailscale/cli/configure-kube.go:38]: Recommend adding a shortlink in tailscale-www/tsweb/redirects.txt so we don't hard-code KB 1236.

**Review by DentonGentry — APPROVED** [review]

File: cmd/tailscale/cli/configure-kube.go (hunk 1 above)
Comment (maisem) [reply cmd/tailscale/cli/configure-kube.go:38]: ack

Comment (maisem) [reply cmd/tailscale/cli/configure-kube.go:38]: will do before merging

File: cmd/tailscale/cli/configure-kube.go [hunk 2]
```diff
@@ -26,12 +26,16 @@ func init() {
 
 var configureKubeconfigCmd = &ffcli.Command{
 	Name:       "kubeconfig",
-	ShortHelp:  "Configure kubeconfig to use Tailscale",
+	ShortHelp:  "Connect to a Kubernetes cluster using a Tailscale Auth Proxy",
 	ShortUsage: "kubeconfig <hostname-or-fqdn>",
 	LongHelp: strings.TrimSpace(`
-Run this command to configure your kubeconfig to use Tailscale for authentication to a Kubernetes cluster.
+*** ALPHA; all of this is subject to change ***
+
```
Comment (mayakacz) [inline cmd/tailscale/cli/configure-kube.go:33]:
```suggestion
```

File: cmd/tailscale/cli/configure.go [hunk 3]
```diff
@@ -17,8 +17,10 @@ var configureCmd = &ffcli.Command{
 	Name:      "configure",
 	ShortHelp: "Configure the host to enable more Tailscale features",
```
Comment (mayakacz) [inline cmd/tailscale/cli/configure.go:18]:
```suggestion
	ShortHelp: "[ALPHA] Configure the host to enable more Tailscale features",
```
We don't have guidance written down, but I believe this is consistent with cmd/tailscale/cli/update.go

File: cmd/tailscale/cli/configure.go [hunk 4]
```diff
@@ -17,8 +17,10 @@ var configureCmd = &ffcli.Command{
...
 	LongHelp: strings.TrimSpace(`
-The 'configure' command is intended to provide a way to configure different
-services on the host to enable more Tailscale features.
+*** ALPHA; all of this is subject to change ***
+
```
Comment (mayakacz) [inline cmd/tailscale/cli/configure.go:21]:
```suggestion
```

**Review by mayakacz — APPROVED** [review]
Or whatever changes are needed to make this consistent with `tailscale update`

**Pull request merged**